test/integration.test.js

```diff
@@ -1,142 +1,193 @@
 import test from 'ava';
 import pify from 'pify';
-import SemanticReleaseError from '@semantic-release/error';
 import releaseNotesGenerator from '../lib/index';
-import commits from './helpers/commits';
 
-const cwd = process.cwd();
+const url = 'https://github.com/owner/repo';
+const lastRelease = {gitHead: '1234'};
+const nextRelease = {gitHead: '456', version: '1.0.0'};
 
-test.afterEach.always(() => {
-  process.chdir(cwd);
-});
-
-test.serial('Use "conventional-changelog-angular" by default', async t => {
-  await commits(['fix(scope1): First fix', 'feat(scope2): Second feature']);
-  const changelog = await pify(releaseNotesGenerator)({});
+test('Use "conventional-changelog-angular" by default', async t => {
+  const commits = [
+    {hash: '111', message: 'fix(scope1): First fix'},
+    {hash: '222', message: 'feat(scope2): Second feature'},
+  ];
+  const changelog = await pify(releaseNotesGenerator)(
+    {},
+    {pkg: {repository: {url}}, lastRelease, nextRelease, commits}
+  );
 
+  t.regex(changelog, new RegExp(`<a name="${nextRelease.version}"></a>`));
```

Comment:
I think the assertions would be easier to read without using the variables. They shouldn't change anyway, I'd suggest to change it to 

```js
t.regex(changelog, new RegExp('<a name="1.0.0"></a>'));
```

to make it more readable?

Reply:
Done